test/test_core.py

```diff
@@ -2921,6 +2921,15 @@ def test_pthread_run_script(self):
     self.set_setting('EXIT_RUNTIME')
     self.do_runf(test_file('pthread/test_pthread_run_script.c'))
 
+  @node_pthreads
+  def test_pthread_wait32_notify(self):
+    self.do_run_in_out_file_test(test_file('wasm_worker/wait32_notify.c'))
+
+  @node_pthreads
+  @no_wasm2js('https://github.com/WebAssembly/binaryen/issues/5991')
+  def test_pthread_wait64_notify(self):
+    self.do_run_in_out_file_test(test_file('wasm_worker/wait64_notify.c'))
```

Comment:
Maybe it makes sense to move these test files into an `atomics/` directory, to avoid the weirdness of doing a pthreads test on a file with `wasm_workers/` in the name?

Reply:
I have a followup to this PR so move that async atomic stuff..  if its OK with I'll create that new folder and move these tests as part of that change?

Reply:
followup sgtm